test/Installments.ts

```diff
@@ -332,7 +332,7 @@ describe("Installments", () => {
             ).to.be.revertedWith("RC_CannotDereference");
         });
 
-        it("returns 0 for an already closed loan", async () => {
+        it("reverts for an already closed loan", async () => {
```

Comment:
Behavior changed here, thought it better to be more explicit about failure for a closed loan (an active loan could also return 0 in the case there is no interest payment due).